index.html

```diff
@@ -250,7 +250,7 @@ <h1>About Me</h1>
                 
                     <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
```

Comment:
- [ ] You are yet to implement the font styling as requested previously. Kindly do that. Your font is quite different from what we have on the Figma template. Kindly take a look at the highlighted styles for hints on how to style your paragraph.

  Your Implementation | Figma Design
  :---:|:---:
  ![rich-font-his](https://user-images.githubusercontent.com/36542986/160594377-7fb6013e-500d-4093-b61f-ebc96a1ded3f.jpg) | ![rich-font](https://user-images.githubusercontent.com/36542986/160594365-37ba9c93-127e-4a56-8762-3a301f8d40df.jpg)

Reply:
done

Reply:
using "inter" font won't give that style, so I have changed it to "sans-serif;" and it gives the exact  same style.